robot_skills/src/robot_skills/arm/handover_detector.py

```diff
@@ -0,0 +1,63 @@
+import rospy
+import PyKDL as kdl
+
+from robot_skills.robot_part import RobotPart
+from tue_manipulation_msgs.msg import GripperCommandGoal, GripperCommandAction
+
+class HandoverDetector(RobotPart):
+    """
+    Sensor functionality to detect when a handover is taking place
+    """
+    def __init__(self, robot_name, tf_listener, side):
+        """
+        constructor
+
+        :param robot_name: robot_name
+        :param tf_listener: tf_server.TFClient()
+        :param side: string used to identify the sensor
```

Comment:
I guess this should be side  (in stead of sensor) right?

Reply:
no, side is an unfortunate name. Its only purpose here is to find the correct topic to listen to.

Reply:
side is a shit name for this parameter, its only purpose here is to identify the topic it needs to listen to.